<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErosionDilation</class>
 <widget class="QWidget" name="ErosionDilation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QScrollArea" name="imageScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>529</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="image">
         <property name="text">
          <string>Image</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Select Operation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="3">
         <widget class="QLabel" name="structuringElementLabel">
          <property name="text">
           <string>Structuring Element:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <widget class="QComboBox" name="structuringElementComboBox">
          <item>
           <property name="text">
            <string>Rectangular</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Elliptic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cross Shaped</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="erodeRadioButton">
            <property name="text">
             <string>Erode</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="dilateRadioButton">
            <property name="text">
             <string>Dilate</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="autoScaleCheckBox">
          <property name="text">
           <string>Auto Scale</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="kernelSizeLabel">
         <property name="text">
          <string>Kernel Size (2n + 1), max = 21)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="kernelSizeSlider">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>21</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
